### PR TITLE
Implementacion de toastAlert y mejoras en el diseño

### PR DESCRIPTION
**Pull request by LordMario**
## Description
* Se importo libreria de Alertas
* Se implemento libreria de alertas en los botones de peticiones
* Se implemento selectBox para campos como el sexo y pictograma blaco/negro

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Refactoring (no functional changes, no api changes)
- [ ] This change requires a documentation update
- [ ] Other (please describe): 

## How Has This Been Tested?

Describe the tests you've run to verify the changes. Provide instructions for reproduction and indicate any relevant details for your test setup.

e
ee
eee
eeee
eeeee
eeeeee
eeeeeee
eeeeeeee
eeeeeeeee
eeeeeeeeee
eeeeeeeeeee
eeeeeeeeeeee
eeeeeeeeeeeee
eeeeeeeeeeeeee
eeeeeeeeeeeeeee
eeeeeeeeeeeeeeee
eeeeeeeeeeeeeeeee
eeeeeeeeeeeeeeeeee

## Checklist:

- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in areas hard to understand.
- [ ] I have made the corresponding changes to the documentation.
- [x] My changes do not generate new warnings.
- [ ] I have added tests that prove my solution is effective or that my feature works.
- [ ] New and existing unit tests pass locally with my changes.
- [ ] All dependent changes have been merged and published in subsequent modules.
- [x] I have reviewed my code and corrected any spelling mistakes.

### CODE REVIEW

**Review by SamBarFu — CHANGES_REQUESTED** [review]

File: lib/l10n/intl_en.arb [hunk 1]
```diff
@@ -37,7 +37,7 @@
   "Cancelar": "Cancelar",
   "Actualizar": "Actualizar",
   "Si_Eliminar": "Si, Eliminar",
-  "Aviso": "Aviso!!!",
+  "Aviso": "Aviso!",
```
Comment (SamBarFu) [inline lib/l10n/intl_en.arb:40]: Agregar signo al inicio

File: lib/l10n/intl_en.arb [hunk 2]
```diff
@@ -37,7 +37,7 @@
...
   "Esta_seguro_que_desea_eliminar_el_pictograma": "Esta seguro que desea eliminar el pictograma '{nameImage}' de '{nameChild}'",
```
Comment (SamBarFu) [inline lib/l10n/intl_en.arb:41]: Revisar ortografía

File: lib/l10n/intl_en.arb [hunk 3]
```diff
@@ -73,9 +73,21 @@
...
   "Informacion_del_nino": "Información del niño",
   "Si_avanzar": "Si, avanzar",
-  "Esta_seguro_de_avanzar_de_fase_a": "¿Está seguro de avanzar de fase a '{nameChild}'?",
+  "Esta_seguro_de_avanzar_de_fase_a": "¿ Está seguro de avanzar de fase a {nameChild} ?",
   "Editar_observaciones": "Editar observaciones",
   "Quitar_actividad": "Quitar actividad",
   "Editar_observaciones_del_nino": "Editar observaciones del niño",
-  "Eliminar_actividad_asignada_del_nino": "Eliminar actividad asignada del niño"
+  "Eliminar_actividad_asignada_del_nino": "Eliminar actividad asignada del niño",
+  "Avance_de_fase_exitosa": "Avance de fase exitosa!",
```
Comment (SamBarFu) [inline lib/l10n/intl_en.arb:81]: agregar signo de admiración al inicio

File: lib/l10n/intl_en.arb [hunk 4]
```diff
@@ -73,9 +73,21 @@
...
+  "Se_avanzo_a_la_fase": "Se avanzó a la fase {numFase} a {nameChild}",
+  "Si_actualizar": "Si, actualizar",
+  "Esta_Seguro_de_actualizar_los_datos": "¿Está seguro de actualizar los datos?",
+  "Actualizado_con_exito": "Actualizado con éxito!",
```
Comment (SamBarFu) [inline lib/l10n/intl_en.arb:85]: agregar signo de admiración al inicio

File: lib/l10n/intl_en.arb [hunk 5]
```diff
@@ -73,9 +73,21 @@
...
+  "Informacion_del_nino_actualizada": "Información del niño actualizada",
+  "Si_salir": "Si, Salir",
+  "Esta_seguro_de_salir_de_la_edicion": "¿Está seguro de salir de la edición?",
+  "Informacion_personal_actualizada": "Información personal actualizada",
+  "Esta_seguro_de_actualizar_el_pictograma": "¿Está seguro de actualizar el pictograma {namePictogram}?",
+  "Se_actualizo_correctamente_el_pictograma_personalizado": "Se actualizo correctamente el pictograma personalizado: {namePictogram}",
```
Comment (SamBarFu) [inline lib/l10n/intl_en.arb:91]: Revisar ortografía

File: lib/l10n/intl_en.arb [hunk 6]
```diff
@@ -73,9 +73,21 @@
   "Logros": "Logros",
   "Informacion_del_nino": "Información del niño",
   "Si_avanzar": "Si, avanzar",
-  "Esta_seguro_de_avanzar_de_fase_a": "¿Está seguro de avanzar de fase a '{nameChild}'?",
+  "Esta_seguro_de_avanzar_de_fase_a": "¿ Está seguro de avanzar de fase a {nameChild} ?",
```
Comment (SamBarFu) [inline lib/l10n/intl_en.arb:76]: Quitar espacios en los signos

**Review by SamBarFu — CHANGES_REQUESTED** [review]

File: lib/l10n/intl_en.arb (hunk 5 above)
Comment (LordMario) [reply lib/l10n/intl_en.arb:91]: Que es lo malo?

File: lib/generated/intl/messages_en.dart [hunk 7]
```diff
@@ -20,28 +20,41 @@ typedef String MessageIfAbsent(String messageStr, List<dynamic> args);
...
   String get localeName => 'en';
 
-  static String m0(nameChild) =>
-      "¿Está seguro de avanzar de fase a \'${nameChild}\'?";
+  static String m0(namePictogram) =>
+      "¿Está seguro de actualizar el pictograma ${namePictogram}?";
 
-  static String m1(nameImage, nameChild) =>
+  static String m1(nameChild) =>
+      "¿ Está seguro de avanzar de fase a ${nameChild} ?";
```
[the remark this answers is not in the document]
Comment (LordMario) [reply lib/generated/intl/messages_en.dart]: Listo

File: lib/generated/intl/messages_en.dart [hunk 8]
```diff
@@ -20,28 +20,41 @@ typedef String MessageIfAbsent(String messageStr, List<dynamic> args);
...
+
+  static String m2(nameImage, nameChild) =>
       "Esta seguro que desea eliminar el pictograma \'${nameImage}\' de \'${nameChild}\'";
 
+  static String m3(namePictogram) =>
+      "Se actualizo correctamente el pictograma personalizado: ${namePictogram}";
```
[the remark this answers is not in the document]
Comment (LordMario) [reply lib/generated/intl/messages_en.dart:33]: Listo

File: lib/generated/intl/messages_en.dart [hunk 9]
```diff
@@ -20,28 +20,41 @@ typedef String MessageIfAbsent(String messageStr, List<dynamic> args);
...
-      "¿Está seguro de avanzar de fase a \'${nameChild}\'?";
+  static String m0(namePictogram) =>
+      "¿Está seguro de actualizar el pictograma ${namePictogram}?";
 
-  static String m1(nameImage, nameChild) =>
+  static String m1(nameChild) =>
+      "¿ Está seguro de avanzar de fase a ${nameChild} ?";
+
+  static String m2(nameImage, nameChild) =>
       "Esta seguro que desea eliminar el pictograma \'${nameImage}\' de \'${nameChild}\'";
```
[the remark this answers is not in the document]
Comment (LordMario) [reply lib/generated/intl/messages_en.dart]: Listo

File: lib/generated/intl/messages_en.dart [hunk 10]
```diff
@@ -20,28 +20,41 @@ typedef String MessageIfAbsent(String messageStr, List<dynamic> args);
...
             MessageLookupByLibrary.simpleMessage("Agregar pictograma"),
         "Anadir_nuevo_pictograma_personalizado":
             MessageLookupByLibrary.simpleMessage(
                 "Añadir nuevo pictograma personalizado"),
+        "Avance_de_fase_exitosa":
+            MessageLookupByLibrary.simpleMessage("Avance de fase exitosa!"),
         "Avanzar_de_fase":
             MessageLookupByLibrary.simpleMessage("Avanzar de fase"),
-        "Aviso": MessageLookupByLibrary.simpleMessage("Aviso!!!"),
+        "Aviso": MessageLookupByLibrary.simpleMessage("Aviso!"),
```
[the remark this answers is not in the document]
Comment (LordMario) [reply lib/generated/intl/messages_en.dart]: Listo

File: lib/l10n/intl_en.arb (hunk 1 above)
Comment (LordMario) [reply lib/l10n/intl_en.arb]: Listo

File: lib/l10n/intl_en.arb (hunk 2 above)
Comment (LordMario) [reply lib/l10n/intl_en.arb]: Listo

File: lib/l10n/intl_en.arb (hunk 3 above)
Comment (LordMario) [reply lib/l10n/intl_en.arb]: Listo

File: lib/l10n/intl_en.arb (hunk 4 above)
Comment (LordMario) [reply lib/l10n/intl_en.arb]: Listo

File: lib/l10n/intl_en.arb (hunk 6 above)
Comment (LordMario) [reply lib/l10n/intl_en.arb]: Listo

File: lib/generated/intl/messages_en.dart [hunk 11]
```diff
@@ -24,10 +24,10 @@ class MessageLookup extends MessageLookupByLibrary {
       "¿Está seguro de actualizar el pictograma ${namePictogram}?";
 
   static String m1(nameChild) =>
-      "¿ Está seguro de avanzar de fase a ${nameChild} ?";
+      "¿Está seguro de avanzar de fase a ${nameChild}?";
 
   static String m2(nameImage, nameChild) =>
-      "Esta seguro que desea eliminar el pictograma \'${nameImage}\' de \'${nameChild}\'";
+      "¿Esta seguro que desea eliminar el pictograma \'${nameImage}\' de \'${nameChild}\'?";
```
Comment (SamBarFu) [inline lib/generated/intl/messages_en.dart:30]: Falta tilde

File: lib/generated/intl/messages_es_NI.dart [hunk 12]
```diff
@@ -24,10 +24,10 @@ class MessageLookup extends MessageLookupByLibrary {
       "¿Está seguro de actualizar el pictograma ${namePictogram}?";
 
   static String m1(nameChild) =>
-      "¿ Está seguro de avanzar de fase a ${nameChild} ?";
+      "¿Está seguro de avanzar de fase a ${nameChild}?";
 
   static String m2(nameImage, nameChild) =>
-      "Esta seguro que desea eliminar el pictograma \'${nameImage}\' de \'${nameChild}\'";
+      "¿Esta seguro que desea eliminar el pictograma \'${nameImage}\' de \'${nameChild}\'?";
```
Comment (SamBarFu) [inline lib/generated/intl/messages_es_NI.dart:30]: Falta tilde

File: lib/l10n/intl_en.arb [hunk 13]
```diff
@@ -37,8 +37,8 @@
   "Cancelar": "Cancelar",
   "Actualizar": "Actualizar",
   "Si_Eliminar": "Si, Eliminar",
-  "Aviso": "Aviso!",
-  "Esta_seguro_que_desea_eliminar_el_pictograma": "Esta seguro que desea eliminar el pictograma '{nameImage}' de '{nameChild}'",
+  "Aviso": "¡Aviso!",
+  "Esta_seguro_que_desea_eliminar_el_pictograma": "¿Esta seguro que desea eliminar el pictograma '{nameImage}' de '{nameChild}'?",
```
Comment (SamBarFu) [inline lib/l10n/intl_en.arb:41]: Falta tilde

**Review by SamBarFu — CHANGES_REQUESTED** [review]

File: lib/l10n/intl_es_NI.arb [hunk 14]
```diff
@@ -37,8 +37,8 @@
   "Cancelar": "Cancelar",
   "Actualizar": "Actualizar",
   "Si_Eliminar": "Si, Eliminar",
-  "Aviso": "Aviso!",
-  "Esta_seguro_que_desea_eliminar_el_pictograma": "Esta seguro que desea eliminar el pictograma '{nameImage}' de '{nameChild}'",
+  "Aviso": "¡Aviso!",
+  "Esta_seguro_que_desea_eliminar_el_pictograma": "¿Esta seguro que desea eliminar el pictograma '{nameImage}' de '{nameChild}'?",
```
Comment (SamBarFu) [inline lib/l10n/intl_es_NI.arb:41]: Falta tilde

**Review by SamBarFu — CHANGES_REQUESTED** [review]

**Review by SamBarFu — APPROVED** [review]

File: lib/l10n/intl_es_NI.arb (hunk 14 above)
Comment (LordMario) [reply lib/l10n/intl_es_NI.arb]: listo

File: lib/l10n/intl_en.arb (hunk 13 above)
Comment (LordMario) [reply lib/l10n/intl_en.arb]: Listo

File: lib/generated/intl/messages_es_NI.dart (hunk 12 above)
Comment (LordMario) [reply lib/generated/intl/messages_es_NI.dart]: Listo

File: lib/generated/intl/messages_en.dart (hunk 11 above)
Comment (LordMario) [reply lib/generated/intl/messages_en.dart]: Listo

**Comment by LordMario** [comment]
> ## Description
> 
>     * Se importo libreria de Alertas
> 
>     * Se implemento libreria de alertas en los botones de peticiones
> 
>     * Se implemento selectBox para campos como el sexo y pictograma blaco/negro
> 
> 
> ## Type of change
> 
> Please delete options that are not relevant.
> 
>     * [ ]  Bug fix (non-breaking change which fixes an issue)
> 
>     * [x]  New feature (non-breaking change which adds functionality)
> 
>     * [x]  Refactoring (no functional changes, no api changes)
> 
>     * [ ]  This change requires a documentation update
> 
>     * [ ]  Other (please describe):
> 
> 
> ## How Has This Been Tested?
> 
> Describe the tests you've run to verify the changes. Provide instructions for reproduction and indicate any relevant details for your test setup.
> 
> e ee eee eeee eeeee eeeeee eeeeeee eeeeeeee eeeeeeeee eeeeeeeeee eeeeeeeeeee eeeeeeeeeeee eeeeeeeeeeeee eeeeeeeeeeeeee eeeeeeeeeeeeeee eeeeeeeeeeeeeeee eeeeeeeeeeeeeeeee eeeeeeeeeeeeeeeeee
> ## Checklist:
> 
>     * [x]  My code follows the style guidelines of this project.
> 
>     * [x]  I have performed a self-review of my own code.
> 
>     * [x]  I have commented my code, particularly in areas hard to understand.
> 
>     * [ ]  I have made the corresponding changes to the documentation.
> 
>     * [x]  My changes do not generate new warnings.
> 
>     * [ ]  I have added tests that prove my solution is effective or that my feature works.
> 
>     * [ ]  New and existing unit tests pass locally with my changes.
> 
>     * [ ]  All dependent changes have been merged and published in subsequent modules.
> 
>     * [x]  I have reviewed my code and corrected any spelling mistakes.

**Comment by LordMario** [comment]
> ## Description
> 
>     * Se importo libreria de Alertas
> 
>     * Se implemento libreria de alertas en los botones de peticiones
> 
>     * Se implemento selectBox para campos como el sexo y pictograma blaco/negro
> 
> 
> ## Type of change
> 
> Please delete options that are not relevant.
> 
>     * [ ]  Bug fix (non-breaking change which fixes an issue)
> 
>     * [x]  New feature (non-breaking change which adds functionality)
> 
>     * [x]  Refactoring (no functional changes, no api changes)
> 
>     * [ ]  This change requires a documentation update
> 
>     * [ ]  Other (please describe):
> 
> 
> ## How Has This Been Tested?
> 
> Describe the tests you've run to verify the changes. Provide instructions for reproduction and indicate any relevant details for your test setup.
> 
> e ee eee eeee eeeee eeeeee eeeeeee eeeeeeee eeeeeeeee eeeeeeeeee eeeeeeeeeee eeeeeeeeeeee eeeeeeeeeeeee eeeeeeeeeeeeee eeeeeeeeeeeeeee eeeeeeeeeeeeeeee eeeeeeeeeeeeeeeee eeeeeeeeeeeeeeeeee
> ## Checklist:
> 
>     * [x]  My code follows the style guidelines of this project.
> 
>     * [x]  I have performed a self-review of my own code.
> 
>     * [x]  I have commented my code, particularly in areas hard to understand.
> 
>     * [ ]  I have made the corresponding changes to the documentation.
> 
>     * [x]  My changes do not generate new warnings.
> 
>     * [ ]  I have added tests that prove my solution is effective or that my feature works.
> 
>     * [ ]  New and existing unit tests pass locally with my changes.
> 
>     * [ ]  All dependent changes have been merged and published in subsequent modules.
> 
>     * [x]  I have reviewed my code and corrected any spelling mistakes.

**Pull request merged**